internal/pkg/template/templates/workloads/partials/cf/nlb.yml

```diff
@@ -130,7 +133,7 @@ NLBCustomDomainAction:
     ServiceName: !Ref WorkloadName
     RootDNSRole: {{ .AppDNSDelegationRole }}
     DomainName:  {{ .AppDNSName }}
-    Aliases: !Split [",", !Ref NLBAliases]
+    Aliases: {{ if .NLB.GetAliases }} {{ fmtSlice .NLB.GetAliases }} {{ else }} [] {{ end }}
```

Comment:
Interesting, why this change? is it some sort of bug fix

Reply:
So previously, we had only one listener and `NLBAliases` used to get value from that single listener. Now when we will have multiple listeners then instead of computing `NLBAliases` prior to coming to this  point we are building it from `GetAliases` method which also makes sure all the aliases that we get as response are unique no matter how customer has mentioned them in the listeners. For example, customer can specify 2 same aliases in 2 different listener rules.